sql/2020/08_Accessibility/common_alt_text_length.sql

```diff
@@ -0,0 +1,33 @@
+#standardSQL
+# Most common lengths of alt text (-1 for none. 2000+ grouped together)
+SELECT
+  client,
+  SUM(COUNT(0)) OVER (PARTITION BY client) AS total_alt_tags,
+
+  alt_length_clipped AS alt_length,
+  COUNT(0) AS occurrences,
+  COUNT(0) / SUM(COUNT(0)) OVER (PARTITION BY client) AS pct_all_occurrences
+FROM (
+  SELECT
+    client,
+    IF(alt_length >= 2000, 2000, alt_length) AS alt_length_clipped
```

Comment:
Totally optional, but I think `LEAST` would be the semantic way to do this in BigQuery
```suggestion
    LEAST(2000, alt_length) AS alt_length_clipped
```

Reply:
ooo i never knew this existed. I'll definitely swap to this instead